### PR TITLE
Mainnet and testnet scripts

### PR DESCRIPTION
**Pull request by zycon91**
scripts to run Testnet and Mainnet nodes

### CODE REVIEW

**Review by charmitro — CHANGES_REQUESTED** [review]
We also want `cmake` & `protobuf-compiler` packages installed.

File: scripts/run_mainnet.sh [hunk 1]
```diff
@@ -4,6 +4,8 @@ set -e
 
 sudo apt update
 sudo apt install -y git clang curl libssl-dev llvm libudev-dev pkg-config
+sudo apt install -y protobuf-compiler
+sudo apt install cmake g++ make
```
Comment (charmitro) [inline scripts/run_mainnet.sh]: Append them on the same command on line 6.

**Review by charmitro — CHANGES_REQUESTED** [review]

File: scripts/run_mainnet.sh (hunk 1 above)
Comment (charmitro) [reply scripts/run_mainnet.sh]: @zycon91

Comment (zycon91) [reply scripts/run_mainnet.sh]: > Append them on the same command on line 6.

sorry for delay. It is done.

File: scripts/run_mainnet.sh [hunk 2]
```diff
@@ -3,9 +3,7 @@
 set -e
 
 sudo apt update
-sudo apt install -y git clang curl libssl-dev llvm libudev-dev pkg-config
-sudo apt install -y protobuf-compiler
-sudo apt install cmake g++ make
+sudo apt install -y git clang curl libssl-dev llvm libudev-dev pkg-config && sudo apt install -y protobuf-compiler && sudo apt install cmake g++ make
```
Comment (charmitro) [inline scripts/run_mainnet.sh]: It should be just:
```shell
sudo apt install -y git clang curl libssl-dev llvm libudev-dev pkg-config protobuf-compiler cmake g++ make
```

**Review by charmitro — CHANGES_REQUESTED** [review]

**Review by charmitro — APPROVED** [review]

**Pull request merged**